Makefile

```diff
@@ -173,15 +173,17 @@ godownloader:
 kustomize-crd:
 	# Create CRD for helm deployment Helm 
 	kustomize build ./definitions/crds > ./charts/kyverno/crds/crds.yaml
-	# Generate install.yaml that have all resources for kyverno
-	kustomize build ./definitions > ./definitions/install.yaml
-	# Generate install_debug.yaml that for developer testing
-	kustomize build ./definitions/debug > ./definitions/install_debug.yaml
 
 # guidance https://github.com/kyverno/kyverno/wiki/Generate-a-Release
 release:
-	kustomize build ./definitions > ./definitions/install.yaml
-	kustomize build ./definitions > ./definitions/release/install.yaml
+	helm template kyverno ./charts/kyverno -n kyverno > ./definitions/stages/release/install.yaml
+	kustomize build ./definitions/stages/release -o ./definitions/install.yaml
+	kustomize build ./definitions/stages/release -o ./definitions/release/install.yaml
+	kustomize build ./definitions -o ./definitions/install.yaml
+	kustomize build ./definitions -o ./definitions/release/install.yaml
```

Comment:
Same here, the following 2 commands are both updating `definitions/release/install.yaml`, do we need both?

`kustomize build ./definitions/stages/release -o ./definitions/release/install.yaml`
`kustomize build ./definitions -o ./definitions/release/install.yaml`